Neos.EventSourcedContentRepository/Classes/Domain/Context/NodeAggregate/NodeAggregate.php

```diff
@@ -182,6 +448,17 @@ public function getStreamName(): StreamName
         return $this->streamName;
     }
 
+    private function traverseEventStream(callable $callback): void
+    {
+        $eventStream = $this->getEventStream();
+        if (!is_null($eventStream)) {
+            foreach ($this->getEventStream() as $eventEnvelope) {
+                $event = $eventEnvelope->getDomainEvent();
+                $callback($event);
+            }
+        }
+    }
+
```

Comment:
am I correct that this method does not fully work yet, i.e. correctly taking Forked Event streams into account?

If that's the case (as I see it currently), is this class actually used so far already? :-)

Reply:
Correct, that's why the node aggregate is currently never used anywhere.

I have a separate class in preparation that replaces the raw event stream and can be used there, but it cannot be populated yet due to known event store issues.

Reply:
add fat comment in the header of the aggregate - then it's fine :)